### PR TITLE
{data}[intel/2021b] netcdf4-python v1.5.5.1 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by rungitta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by rungitta** [comment]
Test report by @rungitta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
login-2.fram.sigma2.no - Linux UNKNOWN UNKNOWN, x86_64, Intel(R) Xeon(R) CPU E5-2683 v4 @ 2.10GHz, Python 2.7.5
See https://gist.github.com/3b469011b4b08b9c91ce6ac3e0a82d11 for a full test report.

**Comment by lexming** [comment]
@boegelbot: please test @ generoso

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 3 (1 easyconfigs in total)
node301.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b184fe6e50d6dc0c0309e391f8ecc014 for a full test report.

File: easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.5.1-intel-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'netcdf4-python'
+version = '1.5.5.1'
```
Comment (lexming) [inline easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.5.1-intel-2021b.eb:4]: This version is from 2020, it is more interesting to have the most recent v1.5.8 in `intel/2021b`. Could please update this PR?

**Review by lexming — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.5.1-intel-2021b.eb (hunk 1 above)
Comment (rungitta) [reply easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.5.1-intel-2021b.eb:4]: Sloppy of me not checking later versions before making the PR.. However, I see that latest version if `1.5.7` and that it got merged four days ago:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.7-intel-2021b.eb

Should we just close this one down then?

Comment (lexming) [reply easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.5.1-intel-2021b.eb:4]: Sure, if 1.5.7 works for you then we can close this PR. Thanks for your contribution @rungitta !

**Comment by lexming** [comment]
Superseded by #14890

**Pull request closed without merging**